lightningd/lightningd.h

```diff
@@ -276,7 +276,7 @@ struct lightningd {
 	int *exit_code;
 
 	/* The round-robin list of channels, for use when doing MPP.  */
```

Comment:
Comment deserves rewording?

Reply:
I left it because the spirit is correct.